linera-client/src/client_options.rs

```diff
@@ -30,34 +30,34 @@ use crate::{
 };
 
 #[derive(Debug, thiserror::Error)]
-pub enum Error {
+pub enum OptionError {
```

Comment:
Not sure `Error` was problematic. What do others think?

Reply:
I prefer `Error`. That's also what the standard library does, e.g with `io::Error` and `fmt::Error`. (If it's used a lot, we could also define the corresponding `Result` type in each case.)